### ISSUE TITLE
ver2.1.0 Error

### ISSUE THREAD

**Issue by damare1** [iholston/lol-bot#152]
![deaa](https://github.com/iholston/lol-bot/assets/144917228/6e1519fc-5f1a-4c5d-8f4c-dcf7c136f7c6)
What should I do?

**Comment by iholston** [comment]
ohh good catch, im putting out a patch now. did you copy your old json file over to the new location?

**Comment by iholston** [comment]
v2.2.1 should work with the old account.json file if you just copy it into the new file location.

**Comment by konoha1231** [comment]
i downloaded version 2.1.1 of the lolbot.exe file and tried running it again but it still doesn't work
![Uploading s.PNG…]()

**Comment by konoha1231** [comment]
![s](https://github.com/iholston/lol-bot/assets/151894058/dbd74f81-8d0b-4c2b-96f8-ec1d12aa5105)

**Comment by damare1** [comment]
![23](https://github.com/iholston/lol-bot/assets/144917228/cbabc211-3835-49b2-9d2a-ea7bd540c383)
I downloaded ver.2.1.1 but same error

**Comment by iholston** [comment]
oh okay, i think i see. Your accounts must be manually transferred to the new account location.
 You can either:
 - use the gui and manually add each account 
 - [easier] copy the old 'lolbot/resources/accounts.json', select 'show in file explorer' on the accounts tab, and paste into that folder

let me know if one of these works for you

**Comment by damare1** [comment]
![aw](https://github.com/iholston/lol-bot/assets/144917228/16e6516a-7679-4448-ba1b-9c36fa09c58d)
I copied and pasted lolbot/resources/accounts.json from ver 2.0.0, but the above error.
Once I quit the program, I could not start it.

**Comment by damare1** [comment]
![awwaa](https://github.com/iholston/lol-bot/assets/144917228/5f91a16e-7d21-4a69-962b-f8ab1a1a5c0d)
I added the account manually and it worked.

**Comment by iholston** [comment]
No, you don't have to it should work from now on. The account processing within the bot was updated to work with .exe so it should work after this one time change. Sorry about that.
